### PR TITLE
Added Wiki cmdlets

### PR DESCRIPTION
**Pull request by gm0d**
# PR Summary

Added:
- Add-VSTeamWiki
- Get-VSTeamWiki
- Remove-VSTeamWiki

to interact with provisioned and code wiki

## PR Checklist

- [x] [Write Help](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-help)
- [x] [Write Unit Test](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-unit-test)
- [x] [Update CHANGELOG.md](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#update-changelogmd)

### CODE REVIEW

**Comment by SebastianSchuetze** [comment]
@gm0d thanks for the PR. I can't commit anything to your PR. Either you fix the mistake yourself or you give me [permission to commit to your PR](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

You have a sample called refenced 'SampleFiles/Get-VSTeamWiki.json' but in your repo it is named 'Get-VsTeamWiki.json'. Mac and Linux are case sensitive. Please renamed the file to get the build run successfully.

File: Source/Classes/Versions.cs [hunk 1]
```diff
@@ -155,5 +160,7 @@ public static string GetApiVersion(APIs service)
       public static string Graph { get; set; } = "";
       public static string Policy { get; set; } = "3.0";
       public static string Processes { get; set; } = "";
+
```
Comment (SebastianSchuetze) [inline Source/Classes/Versions.cs]:
```suggestion
```

remove the empty line

File: Tests/function/tests/Remove-VSTeamWiki.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      It 'project wiki, by id, should unpublish wiki' {
+         ## Act
+         Remove-VSTeamWiki -WikiId $FakeProjectWikiId
+         
+         ## Assert
+         Should -Invoke Remove-VSTeamGitRepository -Exactly -Times 1 -Scope It -ParameterFilter {
+            $Id -eq $FakeProjectWikiId
+         }
+      }
+   }
```
Comment (SebastianSchuetze) [inline Tests/function/tests/Remove-VSTeamWiki.Tests.ps1:91]: Missing a tests to trigger your exception, when the given wiki is not found. Could you please test the exception as well?

**Review by SebastianSchuetze — CHANGES_REQUESTED** [review]
Very nice PR! It is good to see when good structured code and tests are written like the rest of the code in the module.

Just two minor changes would be nice before we can merge it.

File: Source/Classes/Versions.cs (hunk 1 above)
Comment (gm0d) [reply Source/Classes/Versions.cs]: fixed

File: Tests/function/tests/Remove-VSTeamWiki.Tests.ps1 (hunk 2 above)
Comment (gm0d) [reply Tests/function/tests/Remove-VSTeamWiki.Tests.ps1:91]: added test case for invalid Wiki Id

**Review by SebastianSchuetze — APPROVED** [review]
@gm0d Looks good. Just one thing. Could you pull the latest changes from our trunk. There was a Script Analyzer problem that I fixed. When the build is successfully running through, then I can merge it.

**Comment by SebastianSchuetze** [comment]
Never mind. Build was successful. So I am merging.

**Pull request merged**